packages/datagateway-search/src/searchPageTable.test.tsx

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+import { StateType } from './state/app.types';
+import { Provider } from 'react-redux';
+import { MemoryRouter } from 'react-router';
+
+import SearchPageTable from './searchPageTable';
+
+import { mount as enzymeMount } from 'enzyme';
+import { createMount } from '@material-ui/core/test-utils';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { initialState } from './state/reducers/dgsearch.reducer';
+import { dGCommonInitialState } from 'datagateway-common';
+
+describe('SearchPageTable', () => {
+  let mount: typeof enzymeMount;
+  let state: StateType;
+
+  beforeEach(() => {
+    mount = createMount();
+
+    state = JSON.parse(
+      JSON.stringify({ dgsearch: initialState, dgcommon: dGCommonInitialState })
+    );
+  });
+
+  it('renders SearchPageTable correctly', () => {
+    const mockStore = configureStore([thunk]);
+    const wrapper = mount(
+      <div>
+        <SearchPageTable store={mockStore(state)} />
+      </div>
+    );
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('defaults to investigation tab', () => {
+    state.dgsearch = {
+      searchText: '',
+      text: '',
+      selectDate: {
+        startDate: null,
+        endDate: null,
+      },
+      checkBox: {
+        dataset: true,
+        datafile: true,
+        investigation: true,
+      },
+      tabs: {
+        datasetTab: true,
+        datafileTab: true,
+        investigationTab: true,
+      },
+      requestReceived: true,
+      searchData: {
+        dataset: [],
+        datafile: [],
+        investigation: [],
+      },
+    };
```

Comment:
Again, we don't need to completely override the state each time - instead just the specific properties needed to change for the specific test, so either use object spread or simple `state.dgsearch.selectDate = xyz` statements. Makes it easier to see what is being changed in the state for each test